rclcpp_components/src/component_container_isolated.cpp

```diff
@@ -26,10 +26,15 @@ int main(int argc, char * argv[])
   rclcpp::init(argc, argv);
   // parse arguments
   bool use_multi_threaded_executor{false};
+  bool use_events_executor{false};
   std::vector<std::string> args = rclcpp::remove_ros_arguments(argc, argv);
   for (auto & arg : args) {
-    if (arg == std::string("--use_multi_threaded_executor")) {
+    if (arg == std::string("--use_multi_threaded_executor") ||
```

Comment:
Since using underscores in command flags is very uncommon. I added the new option with dashes and allowed for the same here.

Perhaps in the future we can deprecate the original flag based on discussion here.